pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/InPredicateAstNode.java

```diff
@@ -92,7 +95,7 @@ public FilterQueryTree buildFilterQueryTree() {
       throw new Pql2CompilationException("IN predicate has no identifier");
     }
 
-    Set<String> values = new HashSet<>();
+    List<String> values = new ArrayList<>();
```

Comment:
Why changing this to list? We can dedup with Set

Reply:
will change back

Reply:
I want to maintain order so that we can compare the old and new broker request. If deduping is needed, we can use LinkedHashSet